libc/src/assert/assert.h

```diff
@@ -1,3 +1,4 @@
+// NOLINT(llvm-header-guard)
```

Comment:
Note to reviewers: It seems like this MUST be the first line FWICT.

Reply:
That's silly, considering that every single file in LLVM starts with a header comment. I'd open an issue personally.

Reply:
agreed, but there's not much to do for the moment. Perhaps add a todo to move this when possible?

Reply:
Done in https://github.com/llvm/llvm-project/pull/83334/commits/8ef16ad9a4cde241f5c482609ff6549e2b2f7df4 (https://github.com/llvm/llvm-project/issues/83339)